SeleniumWebDriver/SeleniumWebDriver/TutByLoginTest/Tests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Firefox;
+using SeleniumWebDriver.TutByLoginTest.Pages;
+
+namespace SeleniumWebDriver.TutByLoginTest
+{
+    [TestFixture]
+    public class Tests
+    {
+        [TestCase("seleniumtests@tut.by", "123456789zxcvbn", "Selenium Test", "")]
+        [TestCase("seleniumtests2@tut.by", "123456789zxcvbn", "Selenium Test", "")]
+
+        public void TestMethod(string login, string password, string username, string loggedInUser)
+        {
+            IWebDriver _driver = new FirefoxDriver();
+         
+            //create new login page
+            LoginPage loginPage = new LoginPage(_driver);
+
+            //enter login and password
+            loginPage.LogIn(login, password);
+
+            //go to MainPage
+            MainPage mainPage = loginPage.OpenMainPage();
+
+            //get logged in username
+            mainPage.GetUserName(ref loggedInUser);
+
+            //compare usernames
+            Assert.AreEqual(username, loggedInUser);
```

Comment:
Please, don't forget to add message parameter for assert methods to get more details information about error in case test fail

Reply:
fixed with "fd87310"